generate_compose/odcs_requester.py

```diff
@@ -28,6 +28,21 @@ def __call__(self, compose_configs: ODCSComposesConfigs) -> ODCSRequestReference
             self.odcs.wait_for_compose(compose["id"]) for compose in composes
         ]
 
+        failed_composes = [
+            compose for compose in finished_composes if compose["state_name"] != "done"
+        ]
+
+        if failed_composes:
+            failures = [
+                f"{compose['id']}: state={compose['state_name']} "
+                f"reason={compose['state_reason']}"
+                for compose in failed_composes
+            ]
+            sep = "\n"
+            raise RuntimeError(
+                f"Failed to generate some composes:\n{sep.join(failures)}"
```

Comment:
Is there any benefit for showing which composes were generated successfully as well?

Reply:
For now, I think the failed ones should be enough.